intermediate_source/parametrizations.py

```diff
@@ -132,30 +132,41 @@ def forward(self, X):
 # Inspecting a parametrized module
 # --------------------------------
 #
-# When a module is parametrized, we find that the module has changed in two ways:
+# When a module is parametrized, we find that the module has changed in three ways:
+#
+# 1) ``model.weight`` is now a property
 #
-# 1) It has a new ``module.parametrizations`` attribute
+# 2) It has a new ``module.parametrizations`` attribute
 #
-# 2) The weight has been moved to ``module.parametrizations.weight.original``
+# 3) The unparametrized weight has been moved to ``module.parametrizations.weight.original``
 #
 # |
-# We may observe this first change by printing the module
+# After parametrizing ``weight``, we see that, even though calling ``layer.weight`` returns
```

Comment:
I feel like this is a bit ambiguous.
And I don't think you want to go into the details of showing the property with `__class__` below.

Mentioning that `layer.weight` is now a property that contains the parametrized value of the weight. Sounds simpler.